CD/deploy.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+### On any error the script will fail
+set -xeo pipefail
```

Comment:
set '-e' is a shortcut for '-o errexit'
'-o pipefail' add protection against error happening in any commands forming a pipeline.
otherwise only the last command return code is checked.  So error in other command would
not lead to the script exiting with an error.
ex:  
echo "a" | sed -e -badswitch | cat
echo $?
0